vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp

```diff
@@ -42,6 +42,14 @@ void ReactImageBrush::ResizeMode(react::uwp::ResizeMode value) {
   }
 }
 
+void ReactImageBrush::BlurRadius(float value) {
+  if (m_blurRadius != value) {
+    // divide value by 4 to better match android and ios blur strength
```

Comment:
Might this be a difference in whether the number is being treated as a DIP or PX?  Maybe the math should be using scalefactor instead of a random 4?

Reply:
The magic number is 3. 

BlurAmount is the standard deviation, not the blur radius. 

https://microsoft.github.io/Win2D/html/P_Microsoft_Graphics_Canvas_Effects_GaussianBlurEffect_BlurAmount.htm
"You can compute the blur radius of the kernel by multiplying the standard deviation by 3. The units of both the standard deviation and blur radius are DIPs. A value of zero DIPs disables this effect entirely."